nntrainer/opencl/opencl_program.cpp

```diff
@@ -29,8 +31,8 @@ namespace nntrainer::opencl {
  * @return true if successful or false otherwise
```

Comment:
It would be better to add a comment for `@param binaryCreated`

Reply:
Added in latest commit.